src/internal/docker/cli.go

```diff
@@ -1,38 +1,100 @@
 package docker
 
 import (
+	"context"
 	"fmt"
+	"os"
+	"os/exec"
+	"strings"
 
+	"github.com/mitchellh/go-homedir"
 	yey "github.com/silphid/yey/src/internal"
 )
 
 type CLI struct{}
 
-func (c CLI) Start(ct yey.Context, imageTag, containerName string) error {
-	return fmt.Errorf("not implemented")
+func NewCLI() CLI {
```

Comment:
Question I have been wondering. Do we need an empty type? Do we need a yey.Docker interface that this CLI type needs to implement?

Basically the question becomes, are we injecting the this implementation somewhere? Are we planning on mocking the results of CLI[Method] ? 

Would these tests add value?

I am not sure of this, but I wouldn't mind simply having the docker package export functions that wrap os/exec calls,
and having integration tests against actual docker versions as a first pass.

Food for thought.

Reply:
effectivement, je vois pas le besoin du empty struct ou interface à court terme, juste pour les tests éventuellement, mais on peut l'enlever pour l'instant